Models/Task.cs

```diff
@@ -0,0 +1,40 @@
+﻿
+using System.ComponentModel.DataAnnotations;
+
+namespace Fiks.Models
+{
+    public class Task
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        [StringLength(80)]
+        public string Content { get; set; }
+
+        [Required]
+        public int MaxScore { get; set; }
+
+        [Required]
+        public File ReferenceSolution { get; set; }
```

Comment:
Referenční řešení často neexistuje, když už to kolo běží :rtzw:

Tohle bych jako required nedával, často tam jsou z human důvodů prodlevy